src/Microsoft.Diagnostics.Monitoring.WebApi/Utilities.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Diagnostics.Monitoring.WebApi
+{
+    public class Utilities
+    {
+        public const string ArtifactType_Dump = "dump";
+        public const string ArtifactType_GCDump = "gcdump";
+        public const string ArtifactType_Logs = "logs";
+        public const string ArtifactType_Trace = "trace";
+        public const string ArtifactType_Metrics = "collectmetrics";
+
+        internal static string GenerateDumpFileName()
```

Comment:
Imo, internal classes with public methods are much better for this:
- It is clear the methods are meant to be consumed by any caller but the class as a whole is an implementation detail
- It gives room for true internal methods. Things designed to be called by tests or glue code between classes that shouldn't be called by just anyone
- It makes it very easy to switch to a public api should that ever be desired.